stripe/res/layout/activity_payment_sheet.xml

```diff
@@ -34,14 +38,17 @@
             android:layout_height="wrap_content">
 
             <LinearLayout
+                android:id="@+id/fragment_container_parent"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:orientation="vertical">
+                android:orientation="vertical"
+                android:animateLayoutChanges="true">
 
                 <androidx.fragment.app.FragmentContainerView
                     android:id="@+id/fragment_container"
                     android:layout_width="match_parent"
-                    android:layout_height="wrap_content" />
+                    android:layout_height="wrap_content"
+                    android:name="com.stripe.android.paymentsheet.PaymentSheetLoadingFragment"/>
```

Comment:
nice

Reply:
Seems with this added we no longer need to register `PaymentSheetLoadingFragment` in [PaymentSheetFragmentFactory](https://github.com/stripe/stripe-android/blob/1c82cd0acec4e42951162a35aeb20eada518ca5a/stripe/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetFragmentFactory.kt#L12), as we never transition to it programmatically

Reply:
Right, removed.